lib/xfstests_to_junit.pm

```diff
@@ -0,0 +1,126 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary:  base class for convert xfstests wrapper logs to junit format
+# Maintainer: Yong Sun <yosun@suse.com>
+
+package xfstests_to_junit;
```

Comment:
I don't understand why you create this new package which is quite similar with package `ctcs2_to_junit` that generate duplication. As I suggested, hack and expand package `ctcs2_to_junit` should be meet your requirement and this could be used for not only xfstests but also other kernel tests like btrfs and so on.

Reply:
@mitiao xfstests a little different between qa_test_*, it contain some part inside qa_test_*, but use different openQA base. (xfstests is in tests/xfstests, qa_test_* are in tests/qa_automation).

But I do see a function can be reuse, then I change to create a new function xfstests_generateXML, then no need to create a new file.

Reply:
Your changes are still duplicate. From my search and understanding, function `generateXML` in `ctcs2_to_junit` was implemented for and now is only used for xfstests, not related to qa_test_*. So you don't need create a new function for the purpose.